### PR TITLE
Rw update git ignore

### PR DESCRIPTION
**Pull request by rtwhitfield84**
## Description
Updated the .gitignore to ignore  DS_Store

## Related Issue
No related issue

## Motivation and Context
To keep the DS_sTore file from our repo

## Types of changes
 What types of changes does your code introduce? Put an \`x\` in all the boxes that apply: 
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
 Go over all the following points, and put an \`x\` in all the boxes that apply. 
 If you're unsure about any of these, don't hesitate to ask. We're here to help! 
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,6 +2,7 @@
 __pycache__/
 *.py[cod]
 *$py.class
+**/.DS_Store
```
Comment (asimonia) [inline .gitignore:5]: I 👁  what you did there

**Review by asimonia — APPROVED** [review]
Looks super, Richard 🔑

File: .gitignore (hunk 1 above)
Comment (nchemsak) [reply .gitignore:5]: 👍

**Review by nchemsak — APPROVED** [review]
👍

**Pull request merged**